app/views/admin/document_collection_group_memberships/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: admin_document_collection_groups_path(@collection),
+  } %>
+<% end %>
+<% content_for :page_title, @group.heading %>
+<% content_for :title, @group.heading %>
+<% content_for :context, @collection.title %>
+<% content_for :title_margin_bottom, 4 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      <%= view_on_website_link_for @collection, url: { draft: true }, class: "govuk-link", target: "blank" %>
+    </p>
+
+    <%= render "components/secondary_navigation", {
+      aria_label: "Document collection group navigation tabs",
+      items: secondary_navigation_tabs_items(@group, request.path)
+    } %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-three-quarters">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Documents",
+          margin_bottom: 6,
+        } %>
+      </div>
+
+      <div class="govuk-grid-column-one-quarter">
```

Comment:
I think there should be a more efficient way to deal with the layout requirements here: as it is the "Add document" link breaks across lines strangely and floats below and to the right in smaller viewports. The roughly equivalent layout in the summary card component uses flexbox so this might be the best way to deal with it (I assume we are OK with support for this) otherwise it would need to float and respond to breakpoints which could get a bit awkward.